internal/app/orchestrator/orchestrator.go

```diff
@@ -63,14 +61,21 @@ type orchestrator struct {
 // New instantiates the mapper, cache, upstream client components necessary for
 // the orchestrator to operate and returns an instance of the instantiated
 // orchestrator.
-func New(ctx context.Context, l log.Logger, mapper mapper.Mapper, upstreamClient upstream.Client) Orchestrator {
+func New(ctx context.Context,
+	l log.Logger,
+	mapper mapper.Mapper,
+	upstreamClient upstream.Client,
+	cacheConfig *bootstrapv1.Cache) Orchestrator {
 	orchestrator := &orchestrator{
 		logger:         l.Named(component),
 		mapper:         mapper,
 		upstreamClient: upstreamClient,
 	}
 
-	cache, err := cache.NewCache(cacheMaxEntries, orchestrator.onCacheEvicted, cacheTTL)
+	// Initialize cache.
+	cache, err := cache.NewCache(int(cacheConfig.MaxEntries),
```

Comment:
Rather than performing the int cast, can `MaxEntries` be type int?

Reply:
https://developers.google.com/protocol-buffers/docs/proto#scalar int isn't an available type

Reply:
Got it.